index.html

```diff
@@ -305,23 +305,24 @@ <h3>
           "RESOURCE-TIMING-2#dom-performanceresourcetiming">PerformanceResourceTiming</dfn></code>
           interface:
         </p>
-        <ul data-dfn-for="PerformanceResourceTiming">
-          <li>The <dfn>initiatorType</dfn> attribute MUST return the
-          {{DOMString}} "<code>navigation</code>".
+        <ul>
+          <li>The <dfn for="PerformanceNavigationTiming" id='dom-PerformanceNavigationTiming-workerStart'><code>
+            workerStart</code></dfn> getter steps are to perform the following steps:
+            <ol>
+              <li>Let |workerTiming| be |this|'s [=service worker timing=].
+              <li>If |workerTiming| is null, then return |this|'s
+              <a data-cite="resource-timing-2#dom-performanceresourcetiming-workerstart">workerStart</a>.
+              <li>Return |workerTiming|'s [=service worker timing info/start time=].
```

Comment:
```suggestion
              <li>If |workerTiming| is null, then return |this|'s `workerStart`.
              <li>Return |workerTiming|'s [=service worker timing info/start time=].
```

Reply:
This ignores the "superclass" thing and refers to itself. Misses the point, no? How would you put it in prose?

Reply:
Maybe, "|this|'s prototype's `workerStart`".

Reply:
Done